my_game.py

```diff
@@ -59,6 +68,20 @@ def update(self):
         elif self.right > SCREEN_WIDTH - 1:
             self.right = SCREEN_WIDTH - 1
 
+class Canon(arcade.Sprite):
```

Comment:
Let's add an argument _target_sprite_ to  _init()_. Then, the on_update function in Canon() should move itself to the position of the _target_sprite_.